app/scenes/Document/Shared.tsx

```diff
@@ -31,6 +36,7 @@ export default function SharedDocumentScene(props: Props) {
   const { shareId, documentSlug } = props.match.params;
 
   // ensure the wider page color always matches the theme
+  // TODO: I don't know what this does
   React.useEffect(() => {
     window.document.body.style.background = theme.background;
   }, [theme]);
```

Comment:
@tommoor what does this useeffect do here? it doesn't seem to make a difference whether is commented out or not

Reply:
What the comment says :P – I think it can be seen in the area around the page eg if you scroll past the page edges

Reply:
you'll have to show me sometime, I cannot see any difference, when inspecting `window.body` styles